atest/testdata/test_libraries/LibUsingLoggingApi.py

```diff
@@ -24,9 +24,17 @@ def write_messages_to_console():
     logger.console('in two parts')
     logger.info('To log and console', also_console=True)
 
+
```

Comment:
Adding an empty line here is OK because PEP-8 recommends two empty lines between top-level functions. They should be added consistently in the whole module, though. Using just one empty line here is fine too.